calculator.js

```diff
@@ -5,17 +5,67 @@ const exampleAdditionInput = {
   operation: 'add',
 }
 
+const isInteger = function(input) {
+  if (isNaN(input)) console.log(`"${input}" is not a number`);
+  return parseInt(input);
+};
+
+
 const calculateUserInput = function (error, promptInput) {
   console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
 
   // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
+  // What is promptInput? A parameter of the function
+  // What data type? What does it hold? What does it represent? Data type is the object; it holds key-value pairs and represents our inputs.
+  // How do we read values from it? What syntax? By using brackets or dot notation.
+  // How can we use it? To read the input.
+  // Can we call our existing functions now, inside of this function? Yes.
+
+  let first  = isInteger(promptInput.num1);
+  let second = isInteger(promptInput.num2);
+  let operation = promptInput.operation;
+  let result = 0;
```

Comment:
This is valid! But, could we use `const`? Are these variables being reassigned during the function's execution? Do they persist after the function is executed and the program finishes?